dist/helpers.js

```diff
@@ -93,6 +93,6 @@ exports.renderSvg = async (svg, options) => {
         img.addEventListener("load", onLoad);
         img.addEventListener("error", onError);
         document.body.appendChild(img);
-        img.src = "data:image/svg+xml;charset=utf8," + svg;
+        img.src = "data:image/svg+xml;base64," + btoa(svg);
```

Comment:
I don't understand why the svg string needs to be base64 encoded because of the websocket endpoint. Should it just work the same regardless if you're connected to a local Chromium instance or a remote browser?

Reply:
When running via WebSocket I get the "Malformed SVG" error triggered in `onError`. 

Admittedly, I have no real clue why this only happens when running remotely (my naive assumption would probably be that the WebSocket connection breaks something with the transmitted SVG string/function?). 🤔